### PR TITLE
Updates to the PR checklist template

### PR DESCRIPTION
**Pull request by afeddersen**
The existing checklist questions are great but I'd like to introduce a new format that makes it clear if a question needs to be answered.  We've made some minor mistakes recently and I'm hoping this new format/structure will help us avoid those going forward.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/image.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+## Chainguard Images Pull Request Template
+
+<!-- 
+*** PULL REQUEST CHECKLIST: PLEASE START HERE ***
+
+The image pull request checklist includes 10 sections:
+
+* Every section begins with a heading level 3 (e.g., ### Section One).
+* You are required to check at least one box per section -- no exceptions!
```
[the remark this answers is not in the document]
Comment (afeddersen) [reply .github/PULL_REQUEST_TEMPLATE/image.md:9]: Adding an automated check is a great idea, but I think we'd need to modify the PR workflow to distinguish between different types of PRs (e.g., A PR to create or update an image or a PR to do something more general in the repository).   I don't think we'd want the check to be on every PR, correct?  

By modifying the PR workflow, I mean, for example, that we require a label for PRs when you're adding or updating an image (...and our check would look for that label before enforcing the checklist template).  Or maybe we can add a check for something about adding or updating an image in the PR title.  

If it's ok we could discuss this and add it after this update?

[the remark this answers is not in the document]
Comment (kaniini) [reply .github/PULL_REQUEST_TEMPLATE/image.md:9]: Lets stick to doing the simple thing for now, yes

File: .github/PULL_REQUEST_TEMPLATE/image.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+### Access Control + Authentication
+
+- [ ] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default
+- [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
+- [ ] The image requires a non-standard username or non-standard GID/UID (please explain in the notes).
+
+### Certificates
+
+- [ ] A ca-certs bundle was requested and it's included in the image.
+- [ ] A ca-certs bundle was not requested.
```
[the remark this answers is not in the document]
Comment (afeddersen) [reply .github/PULL_REQUEST_TEMPLATE/image.md]: Removed.

**Review by kaniini — APPROVED** [review]

**Comment by afeddersen** [comment]
WTF?  Ha.  Abort. Abort.

**Pull request closed without merging**